lib/rpmsg/rpmsg_virtio.c

```diff
@@ -93,6 +93,11 @@ static void rpmsg_virtio_return_buffer(struct rpmsg_virtio_device *rvdev,
 				       uint16_t idx)
 {
 	unsigned int role = rpmsg_virtio_get_role(rvdev);
+
+#ifdef VIRTIO_CACHED_BUFFERS
+	metal_cache_invalidate(buffer, len);
```

Comment:
Any reason to use `metal_cache_invalidate` instead of `metal_cache_flush`?
Both seems Ok to me as the buffers are returned after their use...